03.CSS_class_manipulation/src/cssClassManipulationTs.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class cssClass {
```

Comment:
please make 2 implementations (files) of this. 
One using class and one not using class. And make sure they both pass tests.